week2/task6/ex6.cs

```diff
@@ -0,0 +1,48 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Excercise 6");
+var signaler = new Signaler();
+signaler.AddTime(new JupiterTime(1, 20));
+signaler.AddTime(new JupiterTime(2, 20));
+signaler.AddTime(new JupiterTime(3, 20));
+signaler.Inform();
+
+public class Signaler
+{
+    List<JupiterTime> Times = new List<JupiterTime>();
+    public void AddTime(JupiterTime time)
+    {
+        Times.Add(time);
+    }
+
+    public void Inform()
+    {
+        if (Times == null) Console.WriteLine("No timers added yet.");
```

Comment:
Times will never be null as you are instantiating it every time by doing `List<JupiterTime> Times = new List<JupiterTime>();`

A better check here would be for the .Count of the Times